src/test/codegen/try_identity.rs

```diff
@@ -14,7 +14,8 @@ type R = Result<u64, i32>;
 #[no_mangle]
 pub fn try_identity(x: R) -> R {
 // CHECK: start:
-// CHECK-NOT: br {{.*}}
+// FIXME(JakobDegen CHECK-NOT): br {{.*}} . This test was broken by changes to enum deaggregation,
```

Comment:
All of these should get an issue filed, though more generally this test in particular seems quite awkward to me, given that it relies on `-Zunsound-mir-opts`...